presentation/src/main/java/com/nexters/boolti/presentation/screen/ticketing/ChooseTicketBottomSheet.kt

```diff
@@ -76,15 +78,15 @@ fun ChooseTicketBottomSheet(
                 modifier = Modifier
                     .padding(top = 12.dp)
                     .size(45.dp, 4.dp)
-                    .background(Grey70)
-                    .clip(RoundedCornerShape(100.dp)),
+                    .clip(RoundedCornerShape(100.dp))
```

Comment:
```suggestion
                    .clip(CircleShape),
```
내가 새로운 걸 발견해 왔어. 내부적으로는 50% 레디어스 적용하고 있더라구

Reply:
믿음으로 갑니다